src/Tools/dotnet-watch/src/StaticContentHandler.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Buffers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Watcher.Tools
+{
+    public class StaticContentHandler
+    {
+        private static readonly byte[] UpdateCssMessage = Encoding.UTF8.GetBytes("UpdateStaticFile||");
+
+        internal static async ValueTask TryHandleFileAction(BrowserRefreshServer browserRefreshServer, FileItem fileItem, CancellationToken cancellationToken)
+        {
+            var filePath = fileItem.StaticWebAssetPath;
+            var bytesToRent = UpdateCssMessage.Length + Encoding.UTF8.GetMaxByteCount(filePath.Length + 2);
+            var bytes = ArrayPool<byte>.Shared.Rent(bytesToRent);
+
+            try
+            {
+                UpdateCssMessage.CopyTo(bytes.AsSpan());
+                var length = UpdateCssMessage.Length;
```

Comment:
nit: could you make the precalculated length a const or at least make it so you aren't accessing the length property twice?